action_server/src/robocorp/action_server/_new_project.py

```diff
@@ -0,0 +1,20 @@
+import logging
+from ._rcc import get_rcc
+
+
+TEMPLATE_URL = "github.com/robocorp/example-action-server-starter"
+
+log = logging.getLogger(__name__)
+
+
+def create_new_project():
+    try:
+        directory = input("Enter path to create the project: ")
```

Comment:
Maybe call it "name"? I think most of the bootstrap cli tools use this language even though it's a directory in the end.

```suggestion
        directory = input("Name of the project: ")
```